src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
do not reset margin and padding for all (`*`) elements. please reset only for needed tags